Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -156,6 +181,11 @@ class SerializerTest : public ::testing::Test {
     ExpectDeserializationRoundTrip(key, absl::nullopt, read_time, proto);
   }
 
+  void ExpectWatchChangeDeserializationRoundTrip(
```

Comment:
I don't think this function adds any value. I'd remove it and just rely on the new overload of `ExpectDeserializationRoundTrip`. I think uniformity makes tests easier to read.